### DISCUSSION TITLE
Q: “你认为这篇文章怎么样“这几个字是能够修改的吗”&&为什么我的页面只能看见评论数却看不到浏览量 || Q: "What do you think of this article?" Can these words be modified?" && Why can I only see the number of comments on my page but not the number of views

### DISCUSSION THREAD

**Discussion by Yun-Tianming** [walinejs/waline#2039] [Q&A] [answered]
### Describe the bug | 问题描述

[网站](https://blog.luofu.monster/comments/)

### Website URL | 问题网站

https://blog.luofu.monster/comments/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

[Website](https://blog.luofu.monster/comments/)

### Website URL | Problem Website

https://blog.luofu.monster/comments/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
1. https://waline.js.org/cookbook/customize/locale.html#locale-%E9%80%89%E9%A1%B9
2. https://waline.js.org/guide/features/pageview.html#%E5%92%8C%E8%AF%84%E8%AE%BA%E4%B8%80%E8%B5%B7%E4%BD%BF%E7%94%A8

多看看文档

---
1. https://waline.js.org/cookbook/customize/locale.html#locale-%E9%80%89%E9%A1%B9
2. https://waline.js.org/guide/features/pageview.html#%E5%92%8C%E8%AF%84%E8%AE%BA%E4%B8%80%E8%B5%B7%E4%BD%BF%E7%94%A8

Read more documents

**Reply by Yun-Tianming** [reply]
谢谢！

---
Thanks!
